README.md

```diff
@@ -1 +1,78 @@
-#DEVELOPMENT BRANCH
\ No newline at end of file
+# Ruby On Rails PROJECT:FORMS
+# 
+## PROJECT:FORMS - [Odin Project](https://www.theodinproject.com/courses/ruby-on-rails/lessons/forms)
+![Hireable](https://cdn.rawgit.com/hiendv/hireable/master/styles/default/yes.svg) 
+
+This project's intention was to build forms using nearly-pure HTML and then graduating to using the helper methods that Rails provides.
+
+## Built With
+
+- [Ruby](https://www.ruby-lang.org/en/).
+- [Ruby On Rails](https://rubyonrails.org/).
+- [rspec](https://rspec.info/).
+- [Git](https://git-scm.com/).
+- [Rubocop](https://github.com/microverseinc/linters-config/tree/master/ruby). 
+
+### Prerequisites
+
+- [Ruby](https://www.ruby-lang.org/en/).
+- [Ruby On Rails](https://rubyonrails.org/).
+- [Git](https://git-scm.com/).
+
+## Getting Started
+
+- **[Install Ruby on your machine.](https://www.theodinproject.com/courses/ruby-programming/lessons/installing-ruby-ruby-programming)** 
+- **[Install Rails on your machine.](https://www.theodinproject.com/courses/ruby-on-rails/lessons/your-first-rails-application-ruby-on-rails)**
+- **[Install Git on your machine.](https://git-scm.com/book/en/v2/Getting-Started-Installing-Git)**
+- Install Node.js and Yarn. According to the official yarn website, it is recommended to install Yarn through the npm package manager, which comes bundled with Node.js when you install it on your system.
+Once you have npm installed you can run the following both to install and upgrade Yarn:
+```
+npm install --global yarn
+```
+
+Once you have npm installed you can run the following both to install and upgrade Yarn:
+### Optional
+
+- [Rubocop](https://github.com/microverseinc/linters-config/tree/master/ruby). 
+- [rspec](https://rspec.info/).
+
+
+### Usage
+
+- Create new folder.
+- Open your terminal.
+- Run ``` git clone https://github.com/pipe2442/re-former.git ```.
+- Enter root folder with command ``` cd re-former ```.
+- In the terminal from the root folder, type ``` rails s ``` and run the browser to display the project. 
```

Comment:
- Before starting the server, doesn't a user need to install dependencies and migrate the database? Add instructions for checking out the `feature/re-former` branch and installing `gems`